stdlib/logging/config.pyi

```diff
@@ -15,9 +16,63 @@ if sys.version_info >= (3, 7):
 else:
     _Path = StrPath
 
+_TKey = TypeVar("_TKey", bound=str)
```

Comment:
Maybe you should just use `str` instead of this, as you already require it to be a string?

Reply:
Exactly, why did I do that? Hahahaha.